boards/arm/nucleo_wb55rg/doc/nucleowb55rg.rst

```diff
@@ -178,6 +178,10 @@ Default Zephyr Peripheral Mapping:
 
 - UART_1 TX/RX : PB7/PB6
 - LPUART_1 TX/RX : PA3/PA2 (arduino_serial)
+- I2C_1_SCL : PB8
```

Comment:
Please add I2C in Supported Features table.

Reply:
Thanks for your precious eyes :)